### PR TITLE
Add enable_loop_unrolling

### PR DESCRIPTION
**Pull request by ahnyoung-paul**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

**Review by ghost — APPROVED** [review]
LGTM

File: inference-engine/thirdparty/clDNN/api/engine.hpp [hunk 1]
```diff
@@ -86,6 +87,7 @@ struct engine_configuration {
         uint16_t n_streams = 1,
         const std::string& kernels_cache_path = "",
         uint16_t n_threads = std::max(static_cast<uint16_t>(std::thread::hardware_concurrency()), static_cast<uint16_t>(1)),
+        bool enable_loop_unrolling = true,
```
Comment (yeonbok) [inline inference-engine/thirdparty/clDNN/api/engine.hpp]: where is this used? I think unrolling config is required only in the IE.

Comment (ahnyoung-paul) [reply inference-engine/thirdparty/clDNN/api/engine.hpp]: remove it. thanks

**Comment by ahnyoung-paul** [comment]
https://github.com/ahnyoung-paul/openvino/pull/14

**Pull request merged**